ImputationPipeline/Validation/ValidateScoring.wdl

```diff
@@ -2,6 +2,11 @@ version 1.0
 
 import "../ScoringPart.wdl" as Scoring
 import "https://raw.githubusercontent.com/broadinstitute/palantir-workflows/main/ImputationPipeline/ScoringPart.wdl" as ScoringMain
```

Comment:
Why no relative import?

Reply:
we specifically want to import the `main` branch version so we can compare it to whatever version is on "this" branch.  Obviously this is a little silly if you are running it on the main branch, but the intended use is really to test out other branches before merging.  I would love to find a better way to do this type of branch comparison in the future.

Reply:
we specifically want to import the main branch version so we can compare it to whatever version is on "this" branch. Obviously this is a little silly if you are running it on the main branch, but the intended use is really to test out other branches before merging. I would love to find a better way to do this type of branch comparison in the future.

Reply:
Oh, that makes a lot of sense